lib/std/jsonutils.nim

```diff
@@ -92,31 +102,36 @@ proc checkJsonImpl(cond: bool, condStr: string, msg = "") =
 template checkJson(cond: untyped, msg = "") =
   checkJsonImpl(cond, astToStr(cond), msg)
 
-template fromJsonFields(a, b, T, keys) =
+template fromJsonFields(a, b, T, keys, opt) =
```

Comment:
optional nit: `s/b/j/` would make it clearer which is the json

Reply:
I'm not sure what you mean? Do you want to change the signature to something like:
```nim
template fromJsonFIelds(obj, json, T, keys, opt)
```

Reply:
ya that would be even clearer

Reply:
Done.